scripts/local_k8s/unittests/test_cluster.py

```
@@ -0,0 +1,360 @@
+            ),
+        ], "Docker-backed kind must use kind load docker-image"
+
+    def test_podman_kind_image_import_saves_archive_before_loading(
+        self,
+        monkeypatch: pytest.MonkeyPatch,
+        preview_config: PreviewConfig,
+    ) -> None:
+        """Verify rootless Podman kind uses an archive load path."""
+        config = replace(preview_config, container_engine="podman", k8s_provider="kind")
+        removed_archives: list[Path] = []
+        archive_path = Path("/tmp/wildside-preview-image.tar")
```

Ruff (0.15.18)

[error] Probable insecure usage of temporary file or directory: "/tmp/wildside-preview-image.tar"

(S108)